react/src/components/NoteList.jsx

```diff
@@ -0,0 +1,23 @@
+import React, {useEffect, useState, useRef} from 'react';
+import Note from './Note';
+
+function List(){
+ const [list, setList] = useState([]);
+ const url = 'http://localhost:8080/api/cards';
```

Comment:
We should have a `config` file to store this variables. To avoid repeating ourselves. When we need to make a call to the API we can simply do `constant url = '{host}/cards'`

Reply:
Good idea! I'll create config file and reach out when changes are ready.